zebra/zebra_dplane.c

```diff
@@ -402,6 +403,19 @@ struct zebra_dplane_provider {
 	TAILQ_ENTRY(zebra_dplane_provider) dp_prov_link;
 };
 
+/* Declare types for list of zns info objects */
+PREDECL_DLIST(zns_info_list);
+
+struct dplane_zns_info {
```

Comment:
What does 'zns' stand for? Is it Zebra Name Space? Or Zebra Netlink Socket? Just a clarification needed for my understanding...

Reply:
it's a shortcut for 'zebra namespace' - there are many places in the code where a `zns` variable name are used for a `struct zebra_ns` (from `zebra_ns.h`).